### ISSUE TITLE
Calculation of current through pzct-02

### ISSUE THREAD

**Issue by mightChamp** [bvarner/PZCT-02-BurdenShift#1]
Can you please explain how o measure current accurately. I got non linear results through it. using openenergymonitor : 
https://github.com/openenergymonitor/EmonLib

What is turn ration inside it?
How you calculate 18ohm burden resistance.

**Comment by bvarner** [comment]
Hi! It’s been a few years since I went through all this, and my math may be a bit off or my memory very fuzzy.

I found this link this evening
https://upcommons.upc.edu/bitstream/handle/2117/26509/IMS-SN2-1.pdf;jsessionid=EF765717C188F3E699561AE7B48E37ED?sequence=1

There’s other stuff out there on how to calculate or size these. Search around.

With the selections I made, my notes say this CT is a 1000:1, and after the resistor, 1mv represents 1ma through the measured line.

I use an ADS1115 for an ADC.

Interestingly enough, I just got some supplies to update this project and clean it up a bit. I planned on starting that in the coming weeks.

Best of luck to you!
